crypto/fipsmodule/rand/rand.c

```diff
@@ -159,6 +174,7 @@ static void CRYPTO_fips_get_from_entropy_source(struct rand_thread_state *state,
   uint8_t *out_entropy, size_t out_entropy_len, int *out_want_additional_input) {
 
 #if defined(FIPS_ENTROPY_SOURCE_JITTER_CPU)
+
```

Comment:
NP: up on line 172 the comment is out of date for passive entropy which doesn't use the state

Reply:
I think it's better just to have a general requirement instead of bi-modal ones.